src/Forms/Form.php

```diff
@@ -644,10 +644,10 @@ public function getValidator()
 
     /**
      * Set the {@link Validator} on this form.
-     * @param Validator $validator
+     * @param null|Validator $validator
      * @return $this
      */
-    public function setValidator(Validator $validator)
+    public function setValidator(?Validator $validator)
```

Comment:
Makes sense to have this param nullable, as there is no default and you should be able to 'unset' any existing validators, as there is no longer a need to always have a validator now that the Form will loop form fields and call validate() on them